pkg/setting/setting.go

```diff
@@ -304,6 +304,9 @@ type Cfg struct {
 
 	DateFormats DateFormats
 
+	// User
+	UserInviteMaxLifetimeDays int
```

Comment:
I'm wondering if this should be made into a duration type field, that is configured via a string such as "2d" or "2h"? We might want to bound the supported lower and upper bound. It sounds to me as if it would make for a more pleasant and granular user experience, although I'm interested in other people's POV.

Reply:
Yes, we use that for other settings (I think).